spec/frameSpec.js

```diff
@@ -0,0 +1,145 @@
+        Scorecard.startGame("Mabon")
+        Scorecard.bowl()
+        //should _getMostRecentFrame() be considered private if used in tests? Wouldn't be called on instances outside of tests.
+        expect(Scorecard.currentGame._getMostRecentFrame().frameFinished).toEqual(true); 
+        expect(Scorecard.currentGame._getMostRecentFrame().strike).toEqual(true);
+      })
+
+      it('correctly sums frame score based on following rolls', () => {
+        spies;
+        Scorecard.startGame("Mabon")
+        Scorecard.bowl();
+        spies.and.returnValue(1);
+        Scorecard.bowl();
+        Scorecard.bowl();
+        expect(Scorecard.currentGame._getPreviousFrame().frameScore).toEqual(12);
+      })
+    })
+
+    describe('spare', () => {
+      let spies;
+      beforeEach(() => {
+        spies = spyOn(window, 'prompt').and.returnValue(5);
+      })
+
+      it('triggers spare boolean', () => {
+        spies;
+        Scorecard.startGame("Mabon");
+        Scorecard.bowl();
+        Scorecard.bowl();
+        expect(Scorecard.currentGame._getMostRecentFrame().spare).toEqual(true);
+      })
+
+      it('correctly sums frame score based on following rolls', () => {
+        spies;
+        Scorecard.startGame("Mabon");
+        Scorecard.bowl();
+        Scorecard.bowl();
+        Scorecard.bowl();
+        expect(Scorecard.currentGame._getPreviousFrame().frameScore).toEqual(15);
+      })
+    })
+
+    describe('10th frame', () => {
+      let spies;
+      beforeEach(() => {
+        spies = spyOn(window, 'prompt').and.returnValue(0);
+      })
+
+      it('calculates final score after the game ends', () => {
+        spies;
+        Scorecard.startGame("Mabon");
+        for (let i = 0; i < 20; i++) {
+          Scorecard.bowl();
+        }
+        expect(Scorecard.bowl()).toEqual(`The game has ended. Your final score was 0.`)
+      })
+
+      it('requests a bonus roll if you score a strike in the final frame', () => {
+        spies;
+        Scorecard.startGame("Mabon");
+        for (let i = 0; i < 18; i++) {
+          Scorecard.bowl();
+        }
+        spies.and.returnValue(10)
+        Scorecard.bowl();
+        Scorecard.bowl();
+        Scorecard.bowl();
+        expect(Scorecard.currentGame.score()).toEqual(30);
+      })
+
+      it('requests a bonus roll if you score a spare in the final frame', () => {
+        spies;
+        Scorecard.startGame("Mabon");
+        for (let i = 0; i < 18; i++) {
+          Scorecard.bowl();
+        }
+        spies.and.returnValue(5)
+        Scorecard.bowl();
+        Scorecard.bowl();
+        Scorecard.bowl();
+        expect(Scorecard.currentGame.score()).toEqual(15);
+      })
+
+      it('calculates a perfect (300) game', () => {
+        spies.and.returnValue(10);
+        Scorecard.startGame("Mabon");
+        for (let i = 0; i < 21; i++) {
+          Scorecard.bowl();
+        }
+        expect(Scorecard.currentGame.score()).toEqual(300);
+      })
+    })
+  })
+})
```

Comment:
These should be in a separate file called "featureSpec".
Good feature tests, though ;)